npc/re/merchants/Extended_Ammunition.txt

```diff
@@ -0,0 +1,525 @@
+		end;
+	case 3:
+		mes "[Mado Gear Merchant]";
+		mes "I will arrange for the consumable packs used by MadoGear to be boxed!";
+		close2;
+		callshop "Box_Exchange_Mechanic", 1;
+		end;
+	case 4:
+		mes "[Mado Gear Merchant]";
+		mes "Do you have something to sell?";
+		close2;
+		callshop "Selling_Items", 2;
+		end;
+	case 5:
+		mes "[Mado Gear Merchant]";
+		mes "You can take a look at other things more calmly.";
+		close;
+	}
+	end;
+	
+OnInit:
+	setunitdata getnpcid(0),UNPC_GROUP_ID,11;
+	end;
+}
+
+//= Talisman Merchant
+//============================================================
+-	script	#over_talisman	HIDDEN_NPC,{
+	mes "[Talisman Merchant]";
+	mes "Hello. Did you come to see the spirits?";
+	next;
+	switch ( select( "Buy Talisman", "Talisman Bundle Exchange", "Cancel" )) {
+	case 1:
+		mes "[Talisman Merchant]";
+		mes "You'll like it.";
+		close2;
+		callshop "Talisman_Merchant", 1;
+		end;
+	case 2:
+		mes "[Talisman Merchant]";
+		mes "We also exchange for bundles.";
+		close2;
+		callshop "Talisman_Bundle", 1;
+		end;
+	case 3:
+		mes "[Talisman Merchant]";
+		mes "Thank you. Come again.";
+		close;
+	}
+	end;
+	
+OnInit:
+	setunitdata getnpcid(0),UNPC_GROUP_ID,12;
+	end;
+}
+
+//= Basic Shops
+//============================================================
+
+//= Archer
+-	shop	Arrow_Archer	HIDDEN_NPC,1750:-1,1770:-1,1751:-1,1752:-1,1754:-1,1755:-1,1756:-1,1757:-1,1762:-1,1065:-1,7940:-1
+
+//= Gunslinger
+-	shop	Weapons_Gunslinger	HIDDEN_NPC,13120:-1,13122:-1,13189:-1,13195:-1,13192:-1,13193:-1,13194:-1,13197:-1,13198:-1,28200:-1,28201:-1
+-	shop	Ammunition_Gunslinger	HIDDEN_NPC,13200:-1,13221:-1,13228:-1,13229:-1,13230:-1,13231:-1,13232:-1,13222:-1,13215:-1,13216:-1,13217:-1,13218:-1,13219:-1,13220:-1,7663:-1,7665:-1,7664:-1,25187:-1
+
+//= Ninja
+-	shop	Weapons_Ninja	HIDDEN_NPC,13258:-1,13257:-1,13255:-1,13256:-1,13259:-1,13294:-1,6512:-1,6513:-1,6514:-1,13250:-1,13251:-1,13252:-1,13253:-1,13254:-1,7521:-1,7522:-1,7523:-1,7524:-1
+
+//= Merchant
+-	shop	Weapons_Mechanic	HIDDEN_NPC,2139:-1,2800:-1,2801:-1,2802:-1,2803:-1,2805:-1,2806:-1,6186:-1,2808:-1,2807:-1
+-	shop	Ammunition_Mechanic	HIDDEN_NPC,18000:-1,18003:-1,18001:-1,18002:-1,18004:-1,6145:-1,6146:-1,12394:-1,12393:-1,12392:-1,6147:-1,2804:-1,2809:-1,2810:-1
+
+//= Talisman
+-	shop	Talisman_Merchant	HIDDEN_NPC,1000563:30:1,101117:15000:1
+
+//= Sale Shop
+-	shop	Selling_Items	HIDDEN_NPC,512:-1
+
+//= Duplicate of Prontera
+//============================================================
+
+prt_in,162,135,5	duplicate(#over_arrow)	Arrow Merchant#over_arrow_0	4_M_ARCHER
+prt_in,162,137,5	duplicate(#over_bullet)	Ammunition Merchant#over_bullet_0	4_F_GUNSLINGER
+prt_in,162,139,5	duplicate(#over_kunai)	Ninja Tool Merchant#over_kunai_0	4_M_NINJA_RED
+prt_in,162,141,5	duplicate(#over_madogear)	Mado Gear Merchant#over_madogear_0	4_TOWER_17
+
+prt_in,162,133,3	duplicate(#over_talisman)	Talisman Merchant#over_talisman_0	4_M_POORSCHOLAR
+
+//= Duplicate of Izlude
```

Comment:
![grafik](https://user-images.githubusercontent.com/3517879/150881323-5c8549cd-4db2-4a86-89ef-e7f57d777b52.png)

Reply:
Did not quite understand!

Reply:
Oh yes!